lib/dogapi/common.rb

```diff
@@ -116,8 +116,10 @@ def request(method, url, extra_params, body, send_json, with_app_key=true)
       resp = nil
       connect do |conn|
         begin
-          current_url = url + prepare_params(extra_params, with_app_key)
+          current_url = url + prepare_params(extra_params)
           req = method.new(current_url)
+          req['DD-API-KEY'] = @api_key
+          req['DD-APPLICATION_KEY'] = @application_key if with_app_key
```

Comment:
The underscore seems to be problematic here - no requests work with it. If I replace it with a hyphen, so `DD-APPLICATION-KEY`, everything seems to work fine.

Reply:
Done. Thanks!